pkg/authorizer/metadata_authorizer.go

```diff
@@ -106,11 +107,11 @@ func (s *MetadataBasedAuthorizer) GetMatchingDbRole(ctx context.Context, tableNa
 
 func (s *MetadataBasedAuthorizer) Configure(tableName string, roleMapping map[string]dbrole.DbRole) {
 	if s.roleMapping == nil {
-		// TRACE("RoleMapping: setting to NEWMAP")
+		TRACE("RoleMapping: setting to NEWMAP")
```

Comment:
I'd say this log statement is not that important. At the end of the function, you're gonna print role mapping in any case. Why print something here? To show that the caller didn't pass a map?

Reply:
there was issue with *metabasedauthorizer vs pass by value which is fixed now. noted can remove it later